moxy-compiler/src/main/java/moxy/compiler/viewstate/ViewStateClassGenerator.kt

```diff
@@ -79,14 +81,13 @@ class ViewStateClassGenerator : JavaFilesGenerator<ViewInterfaceInfo>() {
 
         var commandFieldName: String = decapitalizeString(method.commandClassName)
         var iterationVariableName = "view"
-        val random = Random()
 
         // Add salt if contains argument with same name
         while (method.argumentsString.contains(commandFieldName)) {
-            commandFieldName += random.nextInt(COMMAND_FIELD_NAME_RANDOM_BOUND)
+            commandFieldName += commandFieldName.hashCode() % 10
```

Comment:
I think we need to use integer suffix. In this case each command has a unique name without any loops.

Reply:
It is about clash between user written parameter name and generated name. For example, what will happen, if user type parameter with name view2? We would add integer suffix 2 and generate local variable view2, that would break compilation. Also we dont need any suffixes, if method does not have parameter named view.